deploy/osd-logging/05-role.yaml

```diff
@@ -66,3 +67,50 @@ rules:
   - patch
   - update
   - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: dedicated-admins-openshift-operators-redhat
+  namespace: openshift-operators-redhat
+rules:
+- apiGroups:
+  - operators.coreos.com
+  resources:
+  - installplans
+  verbs:
+  - update
+- apiGroups:
+  - logging.openshift.io
+  resources:
+  - elasticsearch
+  - kibana
+  verbs:
+  - create
+  - delete
+  - deletecollection
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - operators.coreos.com
+  resources:
+  - subscriptions
+  - clusterserviceversions
+  verbs:
+  - "*"
+- apiGroups:
+  - ""
+  resources:
+  - events
+  - namespaces
+  - persistentvolumeclaims
+  - persistentvolumes
```

Comment:
why do they need PV permissions?
does this acutally do something as a Role?

Reply:
it needs to be able to list the PVs if i understood correctly and those can be in this namespace or in the other

Reply:
Iirc pv are namespace agnostic, so putting it here might be confusing (or at least it was for me)

Reply:
PVs are in fact namespace agnostig, PVCs are not.
additionally that role needs the privileges anyways so it doesn't matter if they are namespace scoped.

Reply:
I don't think it makes sense to put them in the role.. what about putting them in the clusterrole?

Reply:
Which one?

Reply:
The pv

Reply:
No, in which clusterrole

Reply:
I didn't see one.. maybe create one for it in this folder?

It's separates the roles but makes.it clean it's a cluster wide resource